dist/edu-benefits-schema.json

```diff
@@ -320,19 +324,31 @@
     "address": {
       "$ref": "#/definitions/address"
     },
+    "email": {
+      "type": "string",
+      "format": "email"
+    },
     "phone": {
       "$ref": "#/definitions/phone"
     },
-    "secondaryPhone": {
+    "mobile": {
       "$ref": "#/definitions/phone"
     },
-    "emergencyContact": {
+    "preferredContactMethod": {
+      "type": "string",
+      "enum": [
+        "mail",
+        "email",
+        "phone"
+      ]
+    },
+    "secondaryContact": {
       "type": "object",
       "properties": {
         "fullName": {
           "$ref": "#/definitions/fullName"
         },
-        "sameAddressAndPhone": {
+        "sameAddress": {
```

Comment:
This is output as "Same address and phone as claimant" in the spool file, and may be an important distinction. Maybe not. So another thing to follow up on.